scripts/update-replica.sh

```diff
@@ -26,4 +26,9 @@ niv update sandbox-launcher-x86_64-linux -a rev=$SHA
 echo "Writing asset sources"
 ./scripts/write-dfx-asset-sources.sh
 
+read -r -p "Update the ledger for e2e tests? (y/n)" yn
+if [[ ! $yn =~ [nN] ]]; then
+    ./scripts/update-nns.bash "$SHA"
```

Comment:
Does this also update nns-canister.patch? I don't see how the patch file is staying updated.

Reply:
I think this is it: https://github.com/dfinity/nns-dapp/blob/main/e2e-tests/scripts/nns-canister.patch